native/src/modules/feedback/Feedback.js

```diff
@@ -46,47 +48,87 @@ const Description = styled(ThemedText)`
   font-weight: bold;
 `
 
+const Heading = styled(ThemedText)`
+  font-size: 16px;
+  text-align: center;
+  padding: 10px 30px 30px;
+`
+
+const HappyIconContainer = styled.Image`
+  margin: 100px auto 10px;
+`
+
+const SadIconContainer = styled.Image`
+  margin: 0px auto 10px;
+`
+
 export type PropsType = {|
   comment: string,
   contactMail: string,
   sendingStatus: SendingStatusType,
   onCommentChanged: (comment: string) => void,
   onFeedbackContactMailChanged: (contactMail: string) => void,
-  isPositiveFeedback: boolean,
+  feedbackOrigin: FeedbackOriginType,
   onSubmit: () => Promise<void>,
   theme: ThemeType,
   t: TFunction
 |}
 
-const FeedbackModal = (props: PropsType) => {
+const Feedback = (props: PropsType) => {
   const renderBox = (): React.Node => {
-    const { theme, t, isPositiveFeedback, comment, contactMail, sendingStatus } = props
+    const { theme, t, feedbackOrigin, comment, contactMail, sendingStatus } = props
+    let title
+    let isSearchFeedback = false
+    switch (feedbackOrigin) {
+      case 'positive':
+        title = t('positiveComment')
+        break
+      case 'negative':
+        title = t('negativeComment')
+        break
+      case 'searchInformationNotFound':
+        title = t('wantedInformation')
+        isSearchFeedback = true
+        break
+      case 'searchNothingFound':
+        title = t('wantedInformation')
+        isSearchFeedback = true
+        break
+    }
 
     if (['idle', 'failed'].includes(sendingStatus)) {
       return (
         <>
-          <Caption theme={theme} title={t('feedback')} />
-
+          {!isSearchFeedback && <Caption theme={theme} title={t('feedback')} />}
+          {feedbackOrigin === 'searchNothingFound' && (
+            <>
+              <SadIconContainer source={SadIcon} />
+              <Heading theme={theme}>{t('nothingFound')}</Heading>
+            </>
+          )}
```

Comment:
I think this maybe should be part of the search component directly as it is not directly related to the feedback but rather just the message appearing if there are no results.

Reply:
Makes sense. Moved it to SearchModal